### DISCUSSION TITLE
Current status of fastfetch-cli org and sponsorship

### DISCUSSION THREAD

**Discussion by Guiorgy** [fastfetch-cli/fastfetch#1041] [General]
Before anything else, I don't know the history of this project, so sorry if I make some wrong assumptions.

Currently, the project is managed under the [fastfetch-cli org](https://github.com/fastfetch-cli), which has only 1 memeber, which is @CarterLi, while the original author @LinusDierheimer seems to be missing from there. However, the [Sponsor this project](https://github.com/sponsors/LinusDierheimer) button leads to the page where you can sponsore _LinusDierheimer_ dirrectly, instead of the organization like in the case of the [curl](https://github.com/curl) project.

I'm just wondering if it would make sense for the org to include _LinusDierheimer_ (unless he's completely retired in the project), and to setup a way to sponsore the organization, so the the money, though little, could be shared among the members.

And as a final note, I by no means want to dismiss the efforts that _LinusDierheimer_ has put into the project, in fact I'd like to thank every contributor for the amazing work done.

**Comment by CarterLi** [comment]
> I'm just wondering if it would make sense for the org to include LinusDierheimer

He is in fastfetch-cli org but he made his profile private.

**Reply by Guiorgy** [reply]
> He is in fastfetch-cli org but he made his profile private.

Didn't know that was a thing.

What about the sponsor part?

**Reply by CarterLi** [reply]
I have no idea how to do it.

**Reply by Guiorgy** [reply]
Github has a guide for that in their [docs](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-organization), though, as I don't represent any org, I obviously don't know any more than that.

**Reply by CarterLi** [reply]
I don't have permission to do that. @LinusDierheimer can you have a check?

**Comment by Guiorgy** [comment]
Update: As of commit [e0296b7](https://github.com/fastfetch-cli/fastfetch/commit/e0296b75f6324f43ca05bef20b2cb430ce934261), both Linus and Li have a url for dontaions. Though, I'd prefer the single Sponsor button in the github repo.

**Reply by CarterLi** [reply]
Seems impossible. Github Sponsor doesn't support my country.

**Reply by Guiorgy** [reply]
:(

**Reply by sr79368142** [reply]
> Seems impossible. Github Sponsor doesn't support my country.

Dear @CarterLi , do you have a WeChat or Alipay sponsorship channel?  You've made great contributions to several projects on GitHub, and Chinese users would be happy to sponsor you if you have either of these options.

**Reply by Guiorgy** [reply]
@sr79368142 As mentioned, he has a link for donations in the README, it's just not GitHub Sponsor but an external one (ko-fi as of writing).

**Reply by sr79368142** [reply]
Dear @Guiorgy  , I saw that README document, it's just that Chinese users are accustomed to using WeChat or Alipay as a payment method, and nowadays the Chinese region is very sensitive to the protection of personal privacy, not too accustomed to using bank cards directly, so I hope that Chinese developers can provide the QR code of WeChat or Alipay as a sponsorship channel is better!

**Reply by CarterLi** [reply]
@sr79368142 Thanks for your suggestion. I added the QR code of WeChat in [the Chinese Readme](https://github.com/fastfetch-cli/fastfetch/blob/dev/README-cn.md)

**Comment by CarterLi** [comment]
Wow, big thanks
